storage/sqlite.go

```diff
@@ -111,8 +111,8 @@ func (s *Sqlite) fillRuleTable(config *config.Config) error {
 				Type:      rule.Key.Type,
 				Name:      rule.Key.Name,
 				Namespace: rule.Key.Namespace,
-				Role:      api.Character_name[int32(api.Character_director)],
-				Character: api.Character_director,
+				Role:      "",
+				Character: api.Character_name[int32(api.Character_director)],
```

Comment:
`Character` in `Rule` structure is of type `api.Character`. How did you do this? It should raise an error